_docs_integrate/01-connector-tutorial.md

```diff
@@ -8,47 +8,173 @@ In this tutorial we go through the basic steps necessary to establish a Relation
 
+The payloads for the requests the will be performed during this tutorial contain placeholders marked with `<...>`. Please replace them with values.
+
 # Prerequisites
 
 -   If you want to use your own Connector for executing the examples
     -   [Install the Connector]({% link _docs_integrate/10-connector-installation.md %})
     -   Make sure the [Sync Module is disabled]({% link _docs_integrate/11-connector-configuration.md %}#sync) (because in this tutorial we will synchronize manually via the HTTP endpoint)
     -   Get the API key configured for the Connector. Ask the person who installed the Connector for it.
--   You need the [Enmeshed App]({% link _pages/use.md %}) installed on your mobile device.
-
--   Diagramm
+-   You need the [Enmeshed App]({% link _pages/use.md %}) installed on your mobile device (App language set to `English`).
 
 # Establishing Relationships
 
 In order to communicate with another Identity, a Relationship to that Identity is required. In this first part of the tutorial you will learn how to establish a Relationship between your Connector and another Identity. In this case the other Identity will be the using the App, but it could be another Connector as well.
 
-## Connector: Create a Relationship Template
+## Connector: Create an Attribute
 
-Start by creating a so called Relationship Template on the Connector. You can do so by calling the `POST /api/v1/RelationshipTemplates/Own` route. Use the following JSON in the request body:
+As a prerequisite we provided an Attribute in the Connector that can be shared in the following step by executing `POST /api/v1/Attributes` with the following payload:
+
+```json
+{
+    "content": {
+        "@type": "IdentityAttribute",
+        "owner": "<your connector's identity address>",
+        "value": {
+            "@type": "DisplayName",
+            "value": "Connector Tutorial"
+        }
+    }
+}
+```
+
+You can query the Connectors Address under the route `/api/v1/Account/IdentityInfo`. If you are using our provided Connector please use `id16RPQxvUC8S5aTrRhs3yrDXq6cLkbbBsTY`.
+{: .notice--info}
+
+{% include rapidoc api_route_regex="^post /api/v1/Attributes$" %}
+
+Remember the `id` of the Attribute that you can find in the response. You will need it in the next step.
+
+## Connector: Test your Requests Validity
+
+In order to make sure the Request and its items are valid you can validate it calling the `POST /api/v1/Requests/Outgoing/Validate` route. Send the Request that will be used later under `onNewRelationship` in the Relationship Template.
```

Comment:
> Send the Request that will be used later under `onNewRelationship` in the Relationship Template.

Ich finde, es ist verwirrend, wenn man sagt, dass man hier "etwas mitschicken soll, was man später benutzt". Das hilft einem kaum weiter. Man weiß ja nicht mal, was "onNewRelationship" ist. Hier wäre es besser, wenn man unterhalb der Payload kurz erklärt, was in der Payload steht. Sprich: "Eine Gruppe mit einem Attribut, was an das Gegenüber geteilt wird, und eine weitere Gruppe mit drei Attributen, die vom Gegenüber angefragt werden".